File: LIFELTD.java
/*package whatever //do not write package name here */

import java.io.*;

class LIFELTD {
	public static void main (String[] args) throws IOException {
	BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
	String a,b,c;
	int t = Integer.parseInt(br.readLine());
	while(t!=0) {
	    a = br.readLine();
	    b = br.readLine();
	    c = br.readLine();

	    char a_arr[] = new char[3];
	    char b_arr[] = new char[3];
	    char c_arr[] = new char[3];

	    a_arr = a.toCharArray();
	    b_arr = b.toCharArray();
	    c_arr = c.toCharArray();

	    if(a_arr[0] == 'l' && b_arr[0] == 'l' && b_arr[1] == 'l') {
	        System.out.println("yes");
	    }
	    else if(a_arr[1] == 'l' && b_arr[1] == 'l' && b_arr[2] == 'l') {
	        System.out.println("yes");
	    }
	    else if(b_arr[0] == 'l' && c_arr[0] == 'l' && c_arr[1] == 'l') {
	        System.out.println("yes");
	    }
	    else if(b_arr[1] == 'l' && c_arr[1] == 'l' && c_arr[2] == 'l') {
	        System.out.println("yes");
	    }
	    else {
	        System.out.println("no");
	    }


	    t--;
	}

	}
}

File: LIFELTD.c
#include<stdio.h>
int main()
{
	int t;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
		char a[3],b[3],c[3];
		scanf("%s",a);
		scanf("%s",b);
		scanf("%s",c);


		if( a[0]=='l' && b[0]=='l' && b[1]=='l')
		{
			printf("yes\n");
			continue;
		}
		if(a[1]=='l' && b[1]=='l' && b[2]=='l')
		{
			printf("yes\n");
			continue;
		}
		if(b[0]=='l' && c[0]=='l' && c[1]=='l')
		{
			printf("yes\n");
			continue;
		}
		if(b[1]=='l' && c[1]=='l' && c[2]=='l')
		{
			printf("yes\n");
			continue;
		}
		printf("no\n");
	}
	return 0;
}
